include/klee/Expr/Expr.h

```diff
@@ -174,7 +174,8 @@ class Expr {
     CmpKindLast=Sge
   };
 
-  unsigned refCount;
+  /// @brief Required by klee::ref-managed objects
+  class ReferenceCounter __refCount;
```

Comment:
Identifiers starting with double underscores are reserved "for implementation" according to C++ standard...